conans/client/manager.py

```diff
@@ -137,6 +139,7 @@ def _load_install_conanfile(self, loader, reference_or_path):
             conanfile = loader.load_virtual([reference_or_path])
         else:
             output = ScopedOutput("PROJECT", self._user_io.out)
+            reference_or_path = make_unicode(reference_or_path)
```

Comment:
Should be converted sooner more close to the command as soon as we know it is a path? It feels a little spread.

Reply:
Yes, I tried so. But then it happens that you have such conversion in 10x more places, I tried to be minimal